src/app/shared/print-options-menu/print-options-menu.service.ts

```diff
@@ -1,16 +1,22 @@
 import { Injectable } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
+import { SettingsDbService } from '../../indexedDb/settings-db.service';
 import { PrintOptions } from '../models/printing';
+import { Settings } from '../models/settings';
 
 @Injectable()
 export class PrintOptionsMenuService {
 
+  settings: Settings;
   printOptions: BehaviorSubject<PrintOptions>;
   showPrintView: BehaviorSubject<boolean>;
   printContext: BehaviorSubject<string>;
   showPrintMenu: BehaviorSubject<boolean>;
-  constructor() {
-    let initPrintOptions: PrintOptions = this.setAll(true);
+  constructor(private settingsDbService: SettingsDbService) {
+    if (!this.settings) {
```

Comment:
this.settings isn't used anywhere and the check isn't needed since we're always grabbing the default print options from global settings